content/en/blog/_posts/2021-08-20-seccomp-default.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Enable seccomp for all workloads with a new v1.22 alpha feature"
+date: 2021-08-20
```

Comment:
```suggestion
date: 2021-08-25
```
Wednesday, Aug 25 is open as a publishing date

Reply:
Changed as suggested :+1: